<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Halaman Kasir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Daftar item yang dibeli</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_item">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>751</width>
     <height>221</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>390</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Detail Transaksi</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Given Money:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Money:</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_total_price">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_change_money">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_simpan_transaksi">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Simpan Transaksi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_given_money">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView_search">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>751</width>
     <height>220</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari barang:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_search">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_select">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>390</y>
     <width>140</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Pilih</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_search">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_hapus">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>120</y>
     <width>140</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus Item</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_search</tabstop>
  <tabstop>button_search</tabstop>
  <tabstop>tableView_search</tabstop>
  <tabstop>button_select</tabstop>
  <tabstop>input_given_money</tabstop>
  <tabstop>button_simpan_transaksi</tabstop>
  <tabstop>tableView_item</tabstop>
  <tabstop>button_hapus</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
